### PR TITLE
Settings page refactor

### PR DESCRIPTION
**Pull request by OSPFNeighbour**
split up settings and profile into different pages. put in a burger menu and cleaned up the style a little.

screenshot 2018-03-05 12 47 59
screenshot 2018-03-05 12 47 15
screenshot 2018-03-02 13 00 14

### CODE REVIEW

**Comment by OSPFNeighbour** [comment]
@sjmcculloch @sdunster review when ready

File: client/src/screens/burger/Index.js [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+    const { navigate } = this.props.navigation;
+    navigate('Profile');
+  }
+
+  logout = () => {
+    this.props.dispatch(logout());
+  }
+
+  about = () => {
+    const DeviceInfoAvailable = DeviceInfo.typeof;
```
Comment (sdunster) [inline client/src/screens/burger/Index.js]: lower case for locals please: `deviceInfoAvailable`

File: client/src/screens/burger/Index.js [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+    }
+
+    installUpdate = () => {
+      codePush.sync({ updateDialog: false, installMode: codePush.InstallMode.IMMEDIATE });
+    }
+
+    render() {
+      return (
+        <Center>
+          <Text />
```
Comment (sdunster) [inline client/src/screens/burger/Index.js]: i really hate this is padded with `<Text />` but I'll take it for now.

File: client/src/screens/burger/Profile.js [hunk 3]
```diff
@@ -306,5 +196,4 @@ export default compose(
   connect(mapStateToProps),
   userQuery,
   updateUserProfileMutation,
-  updateLocationMutation,
-)(Settings);
+)(Profile);
```
Comment (sdunster) [inline client/src/screens/burger/Profile.js]: nice work separating this out

File: client/src/screens/burger/Index.js (hunk 2 above)
Comment (OSPFNeighbour) [reply client/src/screens/burger/Index.js]: Yea, i just left that in place. Thats carry over from original code.

Re Buttons - they are components. I reused Scotts component.

import { Button } from '../../components/Button';

**Pull request merged**